book/src/install.md

```diff
@@ -204,7 +204,13 @@ HELIX_RUNTIME=/home/user-name/src/helix/runtime
 Or, create a symlink in `~/.config/helix` that links to the source code directory:
 
 ```sh
-ln -s $PWD/runtime ~/.config/helix/runtime
+ln -Ts $PWD/runtime ~/.config/helix/runtime
+```
+
+If the above command notes that the directory exist either move `~/.config/helix/runtime` to a new location or delete it, then run the symlink command above again. For example to move it:
+
+```sh
+mv ~/.config/helix/runtime ~/.config/helix/runtime.old
 ```
```

Comment:
I don't think we should include a code sample for this: it's common enough knowledge how to move around directories. Some people run every codeblock without reading the associated text so I'm worried the codeblock will cause more installation problems than it solves.

Reply:
That's a good point, I just couldn't believe the 'simplicity' of my broken install when I found it.  I'll edit the PR.